src/player.cpp

```diff
@@ -4160,14 +4160,14 @@ void player::pause()
     search_surroundings();
 }
 
-void player::shout( std::string msg )
+void player::shout( std::string msg, const std::string &verbal_shout_id )
 {
-    int base = 10;
+    int base = 85;
```

Comment:
> calling NPCs over and over to "herd" them indefinitely to get them into dangerous situations and get them killed.

I can add a cooldown effect to prevent this, like there already is for other npc effects. Would that address this concern?

> It would be pretty useless regardless of range due to how limited in use its function is: stopping neutral NPCs from wandering off. That's pretty much it.

You're forgetting about your feedback from https://github.com/CleverRaven/Cataclysm-DDA/pull/16430. This PR can be used in the problematic situations you described there, mitigating the problems with #16430.

> I'm not convinced this would be a good change.
> 
> NPCs are rather easy to lose. Quest givers can run off, AI can decide to charge into zeds, NPCs can decide to hide in tiny, dark rooms etc.